packages/accounts-controller/CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+### Added
+- Add methods to support ERC-4337 accounts ([#3602](https://github.com/MetaMask/core/pull/3602))
+- Add getAccount action to AccountsController ([#1892](https://github.com/MetaMask/core/pull/1892))
+
+### Changed
+- Bump  `@metamask/keyring-controller`  peer dependency to ^12.1.0 ([#3747](https://github.com/MetaMask/core/pull/3747), [#3810](https://github.com/MetaMask/core/pull/3810))
+- Bump `@metamask/keyring-api` to ^3.0.0 ([#3747](https://github.com/MetaMask/core/pull/3747))
+- Bump `@metamask/snaps-controllers` to ^4.0.0  ([#3747](https://github.com/MetaMask/core/pull/3747))
```

Comment:
Two of these entries are breaking (just noticed that the snaps controller is a peer dependency as well). I've added the prefix and moved them to the top

```suggestion
- **BREAKING:** Bump `@metamask/keyring-controller`  peer dependency to ^12.1.0 ([#3747](https://github.com/MetaMask/core/pull/3747), [#3810](https://github.com/MetaMask/core/pull/3810))
- **BREAKING:** Bump `@metamask/snaps-controllers` peer dependency to ^4.0.0  ([#3747](https://github.com/MetaMask/core/pull/3747))
- Bump `@metamask/keyring-api` to ^3.0.0 ([#3747](https://github.com/MetaMask/core/pull/3747))
```